app/samples/time_travel/prj.conf

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 RIT Launch Initiative
+# SPDX-License-Identifier: Apache-2.0
+
+# CPP
+CONFIG_CPP=y
+CONFIG_REQUIRES_FULL_LIBCPP=y
+CONFIG_STD_CPP20=y
+
+# Devices
+CONFIG_RTC=y
+
+# F-Core
+CONFIG_F_CORE=y
+CONFIG_F_CORE_DEVICE=y
+
+# OS
+CONFIG_POLL=y
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_STACK_USAGE=y
+
+CONFIG_TIMESLICING=y
+CONFIG_TIMESLICE_SIZE=50000
+CONFIG_TIMESLICE_PRIORITY=0
+CONFIG_THREAD_STACK_INFO=y
+
+CONFIG_DYNAMIC_THREAD=y
+# Max stack size for dynamic threads
+CONFIG_DYNAMIC_THREAD_STACK_SIZE=512
+# Number of threads in the dynamic thread pool
+# (Zephyr wants 4 for some reason instead of 2)
```

Comment:
this isn't true any more right? i assume it just got copied from a sample from when this was true but can probably halve it

Reply:
Sorry forgot to push last night LOL